utils/rpms/daos.spec

```diff
@@ -142,6 +142,8 @@ Requires(postun): /sbin/ldconfig
 Requires: libfabric >= 1.11.0
 %{?systemd_requires}
 Obsoletes: cart
+# libunwind for Argobots
+Requires: libunwind
```

Comment:
The argobots package should be installing the libunwind package.
Unless daos directly requires libunwind without argobots, I don't think this dependency should be there.

The spec file and debian equivalent for argobots needs to be updated for the new version and the new dependencies.

Reply:
Since original Argobots packaging is source only, I presume we create our own RPMs for it, so do you mean you will take the action ?? ;-)

Reply:
Yes, [`argobots.spec`](https://github.com/daos-stack/argobots/blob/master/argobots.spec) is where the dependency belongs.  That also needs updating to reflect the [update of version from 1.0 to 7fd19876b77f1f4f5787aed8cd242e60b4d653b6](https://github.com/daos-stack/daos/pull/4102/files#diff-d3984dc6b0c98ea42c422b465e317d22b0adc2c332dc6103254e3a72e0a1b37aR5)

The patch from v.10 to 7fd19876b77f1f4f5787aed8cd242e60b4d653b6 should be added to [`argobots.spec`](https://github.com/daos-stack/argobots/blob/master/argobots.spec) as well as adding the git hash to the [`argobots.spec` `Release:`](https://github.com/daos-stack/argobots/blob/b9a454393b7407e3b49d3db59881c1fcccff3c8e/argobots.spec#L3).

Ultimately every change to `utils/build.config` will require updating an RPM packaging of whatever was being updated in `utils/build.config`.

Reply:
Ok, but correct me if I am wrong, this means to also push a separate PR to this "Argobots packaging" GitHub project/repo, right ??
And I believe, I must also add the "configure --enable-libunwind" step into Argobots.spec too.

Reply:
> Ok, but correct me if I am wrong, this means to also push a separate PR to this "Argobots packaging" GitHub project/repo, right ??

That is correct.

> And I believe, I must also add the "configure --enable-libunwind" step into Argobots.spec too.

Indeed, that is also correct.  Some notes about the [daos-stack/argobots](https://github.com/daos-stack/argobots) updates:
- that package is currently on version 1.0 of argobots: https://github.com/daos-stack/argobots/blob/b9a454393b7407e3b49d3db59881c1fcccff3c8e/argobots.spec#L2
  - but upstream as released 1.0.1 and indeed 1.1a1
  - presumably the git commit you want is actually after 1.1a1
  - if that is so, then you should update the argobots package to 1.1a1 and then calculate the diff between 1.1a1 and the commit you want to use and add that as a `Patch1:` to the specfile
  - you will also need to add `.g7fd1987` to the `Release:  1%{?dist}`: https://github.com/daos-stack/argobots/blob/b9a454393b7407e3b49d3db59881c1fcccff3c8e/argobots.spec#L3
 
You will have to make similar changes to `debian/` in that repo.

I can help you as you get into the details.